src/main/java/com/erigitic/commands/JobCommand.java

```diff
@@ -25,67 +25,264 @@
+            return CommandSpec.builder()
+                    .description(Text.of("Prints out a list of items that reward exp and money for the current job"))
+                    .permission("totaleconomy.command.job.info")
+                    .executor(new Info(totalEconomy))
+                    .arguments(GenericArguments.optional(GenericArguments.string(Text.of("jobName"))))
+                    .build();
+        }
 
-                if (job.getRequirement().isPresent()) {
-                    JobBasedRequirement req = job.getRequirement().get();
+        private TotalEconomy totalEconomy;
 
-                    if (req.permissionNeeded() != null && !player.hasPermission(req.permissionNeeded()))
-                        throw new CommandException(Text.of("You're not allowed to join job \"" + jobName + "\""));
+        // Setup pagination
+        private PaginationService paginationService = Sponge.getServiceManager().provideUnchecked(PaginationService.class);
+        private PaginationList.Builder pageBuilder = paginationService.builder();
+
+        public Info(TotalEconomy totalEconomy) {
+            this.totalEconomy = totalEconomy;
+        }
+
+        @Override
+        public CommandResult execute(CommandSource src, CommandContext args) throws CommandException {
+            TEJobManager teJobManager = totalEconomy.getTEJobManager();
+
+            Optional<String> optJobName = args.getOne("jobName");
+            Optional<TEJob> optJob = Optional.empty();
+
+            if (!optJobName.isPresent() && (src instanceof Player)) {
+                optJob = teJobManager.getPlayerTEJob(((Player) src));
+            }
+
+            if (optJobName.isPresent()) {
+                optJob = teJobManager.getJob(optJobName.get(), false);
+            }
 
-                    if (req.jobNeeded() != null && req.jobLevelNeeded() > teJobManager.getJobLevel(req.jobNeeded().toLowerCase(), player)) {
-                        throw new CommandException(Text.of("You need to reach level " + req.jobLevelNeeded() + " as a " + req.jobNeeded() + " first!"));
-                    }
+            if (!optJob.isPresent()) {
+                throw new CommandException(Text.of(TextColors.RED, "Unknown job: \"" + optJobName.orElse("") + "\""));
+            }
+
+            List<Text> lines = new ArrayList<Text>();
+
+            for (String s : optJob.get().getSets()) {
+                Optional<TEJobSet> optSet = teJobManager.getJobSet(s);
+
+                if (optSet.isPresent()) {
+                    Map<String, List<String>> map = optSet.get().getRewardData();
+                    map.forEach((k, v) -> {
+                        //Add event name
+                        lines.add(Text.of(TextColors.GRAY, TextColors.GOLD, TextStyles.ITALIC, k, "\n"));
+                        //Add targets
+                        v.forEach(id -> {
+                            lines.add(Text.of(TextColors.GRAY, "    ID: ", TextColors.DARK_GREEN, id, "\n"));
```

Comment:
Remove `\n`. A new line is already created it looks like, and having two of them makes it to spaced out.